searchcore/src/vespa/searchcore/proton/server/memoryflush.h

```diff
@@ -34,6 +34,14 @@ class MemoryFlush : public IFlushStrategy
                uint64_t maxMemoryGain_in,
                double diskBloatFactor_in,
                vespalib::duration maxTimeGain_in);
+        bool equal(const Config & rhs) const {
```

Comment:
What is the rationale of not using _operator==_ instead ?

Reply:
Probably too much java lately.

Reply:
Fixed